src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.ts

```diff
@@ -69,6 +71,8 @@ export class RbdListComponent extends ListWithDetails implements OnInit {
   tableStatus = new TableStatusViewCache();
   selection = new CdTableSelection();
   icons = Icons;
+  rbdPools: Array<RbdPool>;
```

Comment:
You may change it to boolean value as just checking if any pools exists or not is enough!
```suggestion
  rbdPoolsExist: boolean;
```